SwrveSDK/src/main/java/com/swrve/sdk/config/SwrveConfigBase.java

```diff
@@ -16,6 +18,20 @@
  * Configuration for the Swrve SDK.
  */
 public abstract class SwrveConfigBase {
+
+    /**
+     * Default Log Tag
+     */
+
+    private final String LOG_TAG = "SwrveConfigBase";
+
+    /**
+     * List of supported stacks
+     */
+    public enum SwrveStack{
+        EU, US, ACTIVISION, FEATURESTACK
```

Comment:
Activision is used by customers, featurestacks are not. We can add ease of featurestack config on the Test App UI.